process/adv-notice.md

```diff
@@ -1,72 +1,47 @@
 ---
-title: How to send advance notice of work to the Advisory Committee
+title: Announcing the start of Charter Refinement
 toc: yes
 ---
 
-This document describes the [Strategy Team](https://www.w3.org/staff/strat/)'s [implementation](#policy) of the [Process Document requirements](https://www.w3.org/policies/process/#WGCharterDevelopment) related to advance notice to the Advisory Committee of charters in development. See the [background](#bg) for the advance notice policy and this document.
+The W3C Process defines a [charter refinement phase](https://www.w3.org/policies/process/drafts/#charter-initiation) to help ensure that draft charters are mission-aligned, reflect community input and consensus, and have been well-socialized and widely reviewed *before* formal review by the Advisory Committee. 
```

Comment:
Per [term](https://www.w3.org/policies/process/drafts/snapshots/2025-05#charter-draft) defined in the Process 2025: s/draft charters/charter drafts/